pkg/operation/botanist/component/extensions/operatingsystemconfig/original/components/docker/component.go

```diff
@@ -78,7 +80,7 @@ func (component) Config(_ components.Context) ([]extensionsv1alpha1.Unit, []exte
 		return nil, nil, err
 	}
 
-	logRotateUnits, logRotateFiles := logrotate.Config(pathLogRotateConfig, "/var/lib/docker/containers/*/*.log", "docker")
+	logRotateUnits, logRotateFiles := logrotate.Config(pathLogRotateConfig, "/var/lib/docker/containers/*/*.log", ContainerRuntime)
```

Comment:
For consistency, you might want to use your new constant also in L69.

Reply:
done